config/initializers/active_support_notifications.rb

```diff
@@ -21,6 +21,10 @@
   ActiveSupport::Notifications.subscribe('product.create', ProductNotifierForColleague.new)
   ActiveSupport::Notifications.subscribe('product.create', ProductNotifierForPracticeWatcher.new)
   ActiveSupport::Notifications.subscribe('came.inquiry', InquiryNotifier.new)
+  ActiveSupport::Notifications.subscribe('question.create', MentorsWatchForQuestionCreator.new)
+  ActiveSupport::Notifications.subscribe('question.update', MentorsWatchForQuestionCreator.new)
+  ActiveSupport::Notifications.subscribe('question.create', AIAnswerCreator.new)
+  ActiveSupport::Notifications.subscribe('question.update', AIAnswerCreator.new)
```

Comment:
同じインスタンスが複数作成されることになっちゃうので下記のような感じがいいかもです。

```suggestion
  mentors_watch_for_question_creator = MentorsWatchForQuestionCreator.new
  ActiveSupport::Notifications.subscribe('question.create', mentors_watch_for_question_creator)
  ActiveSupport::Notifications.subscribe('question.update', mentors_watch_for_question_creator)
  ai_answer_creator = AIAnswerCreator.new
  ActiveSupport::Notifications.subscribe('question.create', ai_answer_creator)
  ActiveSupport::Notifications.subscribe('question.update', ai_answer_creator)
```

Reply:
同一インスタンスを1回だけ作成するように修正いたしました。
コミット：https://github.com/fjordllc/bootcamp/pull/8987/commits/7a08275f609f5fa0c8f8dfc47a4303a0c4806991

※rebaseの影響で該当のレビューコメントが表示されなくなっているようなので、こちらに返信させていただきました。🙇‍♀️